### PR TITLE
Add ajax `index` and `show`

### PR DESCRIPTION
**Pull request by BenLirio**
book/api.js exports
index() returns a promise with all books
show(id) returns a promise of a specific book

### CODE REVIEW

File: assets/scripts/book/api.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+'use strict'
+const url = "https://wdi-library-api.herokuapp.com/books"
```
Comment (syrotech) [inline assets/scripts/book/api.js]: Question: Do we have to require a file from the config and store, where we will set this url, or is this a shortcut that works, as well?

File: assets/scripts/book/api.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const index = function() {
+  return $.ajax({
+    url,
+  })
+}
+
+const show = function(id) {
+  return $.ajax({
+    url: url + '/' + id
+  })
```
Comment (syrotech) [inline assets/scripts/book/api.js:15]: do we need 'Method: 'GET' in both ajax calls?

**Review by syrotech — COMMENTED** [review]
Hi Ben!

File: assets/scripts/book/api.js (hunk 2 above)
Comment (BenLirio) [reply assets/scripts/book/api.js:15]: get is implied by $.ajax()

**Pull request merged**